src/api/flaskr/service/study/utils.py

```
@@ -712,15 +713,17 @@ def __json__(self):
         return {"model_name": self.model_name, "model_args": self.model_args}
 
 
-def get_model_setting(app: Flask, script_info: AILessonScript) -> ModelSetting:
+def get_model_setting(
+    app: Flask, script_info: AILessonScript, status: list[int] = [STATUS_PUBLISH]
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)